docs/docs/pods.md

```diff
@@ -42,47 +41,53 @@ Currently, Marathon pods can only be created and administered via the `/v2/pods/
 
 \* Pods cannot be modified by the `/v2/groups/` endpoint, however. Pods are modified via the `/v2/pods/` endpoint.
 
 ## Networking
-Marathon pods only support the [Mesos containerizer](http://mesos.apache.org/documentation/latest/mesos-containerizer/), which simplifies networking among containers. The containers of each pod instance share a network namespace and can communicate over localhost. 
+Marathon pods only support the [Mesos containerizer](http://mesos.apache.org/documentation/latest/mesos-containerizer/). The Mesos containerizer supports multiple image formats, including Docker.
 
-Containers in pods are created with endpoints. Other applications communicate with pods by addressing those endpoints. If you specify a container network without a name in a pod definition, it will be assigned to this default network.
+The Mesos containerizer simplifies networking by allowing the containers of each pod instance to share a network namespace and communicate over localhost.
+
+If you specify a container network without a name in a pod definition, it will be assigned to this default network. <!-- check the pod def for more info on this -->
+
+If you neeed other applications to communicate with your pod, specify an endpoint in your pod definition. Other applications will communicate with your pod by addressing those endpoints. See [the Examples section](#endpoints) for more information.
 
 In your pod definition you can declare a `host` or `container` network type. Pods created with `host` type share the network namespace of the host. Pods created with `container` type use virtual networking. If you specify the `container` network type, you must also declare a virtual network name in the `name` field. See the [Examples](link) section for the full JSON.
 
 ## Ephemeral Storage
 Containers within a pod share ephemeral storage. You can mount volumes by different names on each container in the pod.
 
 ## Pod Definitions
-Pods are configured via a JSON pod definition, which is similar to an [application definition](http://mesosphere.github.io/marathon/docs/application-basics.html). You must declare the resources required by each container in the pod, even if Mesos is isolating at a higher (pod) level.  See the [Examples](link) section for complete pod definitions.
+Pods are configured via a JSON pod definition, which is similar to an [application definition](http://mesosphere.github.io/marathon/docs/application-basics.html). You must declare the resources required by each container in the pod because Mesos, not Marathon, determines how and when to perform isolation for all resources requested by a pod.
+
+See the [Examples](link) section for complete pod definitions.
 
 ### Secrets
 
 Specify a secret in the `secrets` field of your pod definition. The argument should be the fully qualified path to the secret in the store.
 
 ```
 {
-	"secrets": /fully/qualified/path/
+	"secrets": "/fully/qualified/path/"
```

Comment:
also, perhaps a note for **non-dcos** users that additional deployment steps are needed to use secrets:

1. some secrets plugin implementation
1. enable the `secrets` feature via Marathon command-line flags